CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+- Add the `MakeHeaderValue` trait bound to the `SetRequestHeaderLayer`
```

Comment:
I would also point out that this change concerns the `tower-reqwest` crate for clarity.

Reply:
Sorry for the tardy reply. Crate noted in the changelog.